chapter-09/recipe-03/cxx-example/menu.yml

```diff
@@ -1,2 +1,12 @@
 targets:
   - test
+
+appveyor-vs:
+  definitions:
+    - CMAKE_BUILD_TYPE: 'Release'
+    - CMAKE_CONFIGURATION_TYPES: 'Release'
+
+appveyor-msys:
+  definitions:
+    - CMAKE_BUILD_TYPE: 'Release'
+    - CMAKE_CONFIGURATION_TYPES: 'Release'
```

Comment:
Nitpick: only VS knows how to use `CMAKE_CONFIGURATION_TYPES`. It's however harmless to use this with other generators.

Reply:
OK I shall remove this for lower confusion in future.